Task1qsort.c

```diff
@@ -0,0 +1,105 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+
+void swap(int* a, int* b)
+{
+    int temp = *a;
+    *a = *b;
+    *b = temp;
+}
+
+void printArray (int array[], int length) //length
+{
+    for (int i = 0; i < length; i++)
+    {
+        printf("%d ", array[i]);
+    }
+    printf("\n");
+}
+
+void insertionSort (int array[], int start, int finish) //indexes
+{
+    for (int i = start + 1; i <= finish; i++)
+    {   
+        int currentIndex = i;
+        while (currentIndex > start && array[currentIndex] < array[currentIndex - 1])
+        {
+            swap(&array[currentIndex], &array[currentIndex - 1]);
+            currentIndex -= 1;
+        }
+    }
+}
+
+int partition(int array[], int start, int end) //indexes
```

Comment:
```suggestion
int partition(int array[], int start, int end) //indexes
```
Здесь `end`, в предыдущей функции `finish`. Как-то неединообразненько...